Robot-Tests/KW_Leveys.robot

```diff
@@ -0,0 +1,251 @@
+    Radio Button Should Be Set To               road-width    enabled
+    Wait Until Element Is Visible               ${Leveys_tekstikenttä}
+    Input Text                                  ${Leveys_tekstikenttä}    400    True
+    Click Button                                ${FA_header_Tallenna}
+
+    #Muutoksen peruutus (laatikkotyökalu)
+    Element Should Be Visible                   ${Muokkaustila_Laatikko}
+    Click Element                               ${Muokkaustila_Laatikko}
+    Suorita laatikkovalinta
+    Wait Until Element Is Visible               road-width
+    Click Button                                ${FA_header_Tallenna}
+
+    #Leveyden lisääminen polygontyökalulla
+    Wait Until Element Is Enabled               ${Muokkaustila_PolygonTool}
+    Click Element                               ${Muokkaustila_PolygonTool}
+    Suorita monivalinta
+    Wait Until Element Is Visible               road-width
+    Select Radio Button                         road-width    enabled
+    Radio Button Should Be Set To               road-width    enabled
+    Wait Until Element Is Visible               ${Leveys_tekstikenttä}
+    Input Text                                  ${Leveys_tekstikenttä}    400    True
+    Click Button                                ${FA_header_Tallenna}
+
+    #Muutoksen peruutus (polygontyökalu)
+    Element Should Be Visible                   ${Muokkaustila_PolygonTool}
+    Click Element                               ${Muokkaustila_PolygonTool}
+    Suorita monivalinta
+    Wait Until Element Is Visible               road-width
+    Click Button                                ${FA_header_Tallenna}
+
+
+Usean linkin klikkausvalinta    [arguments]    ${testipaikka}
+    Testin Aloitus
+    Log    Usean leveystiedon lisäämistapojen testaaminen
+    Vaihda Tietolaji                                  ${TL_Leveys_RB}
+    Paikanna osoite                                   ${testipaikka}
+    Zoomaa kartta                                     5  100 m
+    Odota sivun latautuminen
+    Siirry Muokkaustilaan
+    Wait Until Element Is Not Visible                 ${Map_popup}
+    Hold Control And Click Element At Coordinates     ${Kartta}  0  10
+    selenium_extensions.Drag And Drop By Offset       ${Kartta}  200  0
+    Click Element At Coordinates                      ${Kartta}  0  10
+    selenium_extensions.Drag And Drop By Offset       ${Kartta}  -400  0
+    Click Element At Coordinates                      ${Kartta}  0  10
+    Release Control
+
+    Wait Until Element Is Visible                     road-width
+    Select Radio Button                               road-width    enabled
+    Radio Button Should Be Set To                     road-width    enabled
+    Wait Until Element Is Visible                     ${Leveys_tekstikenttä}
+    Input Text                                        ${Leveys_tekstikenttä}    400    True
+    Click Button                                      ${FA_header_Tallenna}
+
```

Comment:
Tännekin laittaisin niitä should tarkistuksia lisäyksen onnistumisen tsekkaamiseksi.

Reply:
Lisätty